<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtTestDemoClass</class>
 <widget class="QDialog" name="QtTestDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtTestDemo</string>
  </property>
  <widget class="QLabel" name="video_main">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>500</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: green;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="video_side">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>500</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: green;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>510</y>
     <width>471</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnStart">
      <property name="text">
       <string>startAll</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnStartMain">
      <property name="text">
       <string>startMain</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnStartLeft">
      <property name="text">
       <string>startLeft</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnStop">
      <property name="text">
       <string>stopAll</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>btnStart</zorder>
  <zorder>btnStop</zorder>
  <zorder>video_side</zorder>
  <zorder>btnStartMain</zorder>
  <zorder>btnStartLeft</zorder>
  <zorder>video_main</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtTestDemo.qrc"/>
 </resources>
 <connections/>
</ui>
